app/Http/Controllers/Api/V1/ItemController.php

```diff
@@ -265,6 +313,17 @@ protected function createQueryBuilder($q, $filter)
                 $builder->filter($colorQuery);
                 continue;
             }
+
+            if ($field === 'authors.name' || $field === 'authors.authority.id') {
+                $authorsBuilder ??= Query::bool();
+                $authorsBuilder->should(
```

Comment:
```suggestion
                $authorsBuilder->must(
```

Just a question. Why is this doing a union instead of an intersection? Is there a specific use case for that?

Reply:
For posterity: It's doing a union because I want to be able to see both authors with and without authority ID in the search list. When I pick a 'authority' author, an ID is added to the filter, when I pick a 'non-authority' author, their name is added.